core/meta/inc/TClass.h

```diff
@@ -131,6 +131,22 @@ friend class ROOT::Internal::TCheckHashRecursiveRemoveConsistency;
                         // in ROOT Meta w/o interpreter information
    };
 
+   class ObjectPtr
```

Comment:
A comment would be helpful describing how / in which context `ObjectPtr` is used.